.environment/ktlint/run-ktlintCheck.sh

```diff
@@ -1,16 +1,14 @@
 #!/usr/bin/env bash
 
 echo "Checking for ktlint code violations in PRIME Router..."
-pushd $PWD > /dev/null
```

Comment:
I'm not sure why this was here -- do you want to change just for cleanup, or is it causing a problem? (I think it's probably fine either way, just curious.)